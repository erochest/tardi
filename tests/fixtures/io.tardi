
use: std/fs
use: std/io

// io/write-file
"""
Hello, world!
Goodbye, cruel world!
""" "temporary-file.txt" write-file

// io/read-file
"temporary-file.txt" read-file

// fs/rm
"temporary-file.txt" rm

// io/open

// io/read
// io/read-line
// io/read-lines
// io/read-all

// io/write
// io/write-line
// io/write-lines
// io/write

// fs/exists?
// fs/mkdir
// fs/rmdir
// fs/ensure-dir
// fs/touch

