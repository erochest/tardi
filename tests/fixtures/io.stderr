"io/write-file"
#t
"io/read-file"
"
Hello, world!
Goodbye, cruel world!
"
#t
"fs/rm"
#t
"fs/exists?"
#f
#t
"io/<writer>"
<writer: "tmp/another-temporary.txt">
"io/file-path>>"
"tmp/another-temporary.txt"
"io/close"
#t
#t
<writer: "tmp/temporary-three.txt">
#t
"fs/truncate"
#t
""
"io/write"
#t
"some text to write"
"io/write-line"
#t
#t
#t
"first line
second line
third line
"
"io/write-lines"
#t
"first line
second line
third line
"
"io/flush"
"io/<reader>"
"io/read"
"line 1
line 2
line 3
"
#t
"io/read-line"
"line 1
"
#t
"io/read-lines"
{ "line 1" "line 2" "line 3" }
#t
"fs/ensure-dir"
#t
#f
